[ao] fixing public v private for quantization_mappings.py

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #86037
* #86036
* #86034
* #86033
* #86032
* #86031
* #86030
* #86029
* #86028
* #86027
* #86026
* __->__ #86025
* #86024
* #86023
* #86022

Summary: no significant changes, just added __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

HDCharles committed:
[ao] fixing public v private for quantization_mappings.py

Summary: no significant changes, just added __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on torch/ao/quantization/quantization_mappings.py:
```diff
@@ -28,6 +28,32 @@
 from torch.nn.utils.parametrize import type_before_parametrizations
 
+__all__ = [
+    "DEFAULT_REFERENCE_STATIC_QUANT_MODULE_MAPPINGS",
+    "DEFAULT_STATIC_QUANT_MODULE_MAPPINGS",
+    "DEFAULT_QAT_MODULE_MAPPINGS",
+    "DEFAULT_DYNAMIC_QUANT_MODULE_MAPPINGS",
+    "DEFAULT_FLOAT_TO_QUANTIZED_OPERATOR_MAPPINGS",
+    "DEFAULT_MODULE_TO_ACT_POST_PROCESS",
+    "DEFAULT_STATIC_SPARSE_QUANT_MODULE_MAPPINGS",
+    "DEFAULT_DYNAMIC_SPARSE_QUANT_MODULE_MAPPINGS",
```
these probably should be implementation details since we have getter functions, maybe add a TODO to remove them from public apis?

HDCharles replied:
yeah, will talk about it Thursday

HDCharles committed:
Update on "[ao] fixing public v private for quantization_mappings.py"


Summary: no significant changes, just added __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for quantization_mappings.py"


Summary: no significant changes, just added __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge
